static/js/components/widgets/MarkdownEditor.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useCallback, useMemo, useRef, useState } from "react"
 import { CKEditor } from "@ckeditor/ckeditor5-react"
-import { editor } from "@ckeditor/ckeditor5-core"
+import { Editor } from "@ckeditor/ckeditor5-core"
```

Comment:
The new version of CKEditor types got rid of `editor` namespace, so now `editor.Editor` becomes `Editor`.